src/backend/main/go.main/facilities/REST/draft.go

```diff
@@ -28,7 +28,12 @@ func (rest *RESTfacility) SendDraft(user_id, msg_id string) (msg *Message, err e
 		log.WithError(resolvErr).Info("[SendDraft] failed to resolve sender's protocol")
 		return nil, errors.New("unknown protocol for sender")
 	}
-
+	// Associate to an existing discussion or create a new one
+	_, err = rest.store.GetOrCreateDiscussion(draft.User_id, draft.Participants)
```

Comment:
you are not embedding discussion_id into Message. Is it on purpose ?

Reply:
No I forget this update. Fixed #55095b1c5abf75f0a199817618b5b22f21a256cd